tests/test-functional/test-functional-fp.h

```diff
@@ -253,8 +255,9 @@ testFunctional::testFp(const std::string& category, PimDataType dataType)
           default: assert(0);
         }
         if (vecDest[i] != expected) {
+          // FP32 DIVISION sometimes(in extreme cases where float precision has limit) gives different results that have only very small difference compared to the number itself.
```

Comment:
I suspect this is because of different sequence of float/double type casting. Could you double check if there is any float/double casting introduced in this testing and during functional simulation?